misc/python/materialize/checks/all_checks/owners.py

```diff
@@ -42,7 +42,7 @@ def _create_objects(self, role: str, i: int, expensive: bool = False) -> str:
         if expensive:
             s += dedent(
                 f"""
-                CREATE SOURCE owner_source{i} FROM LOAD GENERATOR COUNTER (SCALE FACTOR 0.01)
+                CREATE SOURCE owner_source{i} FROM LOAD GENERATOR COUNTER
```

Comment:
Please keep the SCALE FACTORs here with a testdrive-based version check so that we still test that we can migrate a load generator created with a scale factor.